test/WebJobs.Script.Tests/app.config

```diff
@@ -1,39 +1,53 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <connectionStrings>

```

Comment:
Done.

---
In reply to: [56192543](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56192543) [](ancestors = 56192543)